libmscore/cleflist.cpp

```diff
@@ -69,14 +69,16 @@ void ClefList::setClef(int tick, ClefTypeList ctl)
 //    return -1, if no such clef
 //---------------------------------------------------------
 
-int ClefList::nextClefTick(int tick) const
+Fraction ClefList::nextClefTick(const Fraction& tick) const
       {
       if (empty())
-            return -1;
-      auto i = upper_bound(tick+1);
+            return Fraction(-1, 1);
+//      auto i = upper_bound(tick+1);
+//      auto i = upper_bound(TimePosition(tick + Fraction::fromTicks(1)));
+      auto i = upper_bound(TimePosition(tick));
       if (i == end())
-            return -1;
-      return i->first;
+            return Fraction(-1, 1);
```

Comment:
If `-1` means failure, let's introduce one more static constant like `Fraction::ERROR` which specifies `Fraction(-1, 1)` value.

Reply:
...mmh. i would not recommend this as a general design pattern and therefore would not like encourage people to use it by adding special support.

Reply:
I agree that this shouldn't be used in general as there might be times where negative fractions are useful. Could it be done with `Fraction(0, 0)` instead? Obviously it shouldn't be possible to create this fraction normally, but maybe there is a way to create it through `Fraction::INVALID` or something like that.